src/System.Composition.AttributedModel/src/System.Composition.AttributedModel.csproj

```diff
@@ -0,0 +1,49 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <ProjectGuid>{C6257381-C624-494A-A9D9-5586E60856EA}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>System.Composition</RootNamespace>
+    <AssemblyName>System.Composition.AttributedModel</AssemblyName>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <TargetFrameworkProfile>Profile259</TargetFrameworkProfile>
```

Comment:
Is this how we intend to continue building this library? As far as I'm aware this would be the only library that actually builds against a portable profile rather than with our contract assemblies via NuGet. We don't have a way to compile against the set of contracts exposed by this profile (lower versions), so moving to NuGet would prevent us from running on Win8/WP8. Just throwing this out there.